typings/internalBinding/modules.d.ts

```diff
@@ -1,26 +1,30 @@
 export type PackageType = 'commonjs' | 'module' | 'none'
-export type PackageConfig = {
-  pjsonPath: string
+export type RecognisedPackageConfig = {
```

Comment:
```suggestion
export type RecognizedPackageConfig = {
```

This should use en-us locale